src/com/lftechnology/java/training/niraj/utils/FileServices.java

```diff
@@ -0,0 +1,31 @@
+package com.lftechnology.java.training.niraj.utils;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Services related to file processing
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public interface FileServices {
+    /**
+     * Reads content of a file
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @return String
+     * @throws FileNotFoundException
+     * @throws IOException
+     */
+    public String readFile() throws FileNotFoundException, IOException;
+
+    /**
+     * Writes content to a file
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @throws FileNotFoundException
+     * @throws IOException
+     */
+    public boolean writeFile(String s) throws FileNotFoundException, IOException;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.FileNotFoundException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)